File: mk1/mk1/TP/Lista3/Zad1/Pieciokat.java
class Pieciokat extends Figura{
    public float bok;

    Pieciokat(float bok){
        this.bok = bok;
    }

    public float Pole(){
        return (float) ((float) (5*bok*bok)/(4*Math.tan(3.14159/5)));
    }

    public float Obwod(){
        return 5*bok;
    }

    public String Nazwa(){
        return "Pięciokąt";
    }
}

File: mk1/mk1/TP/Lista3/Zad2/Pieciokat.cpp
#include "Pieciokat.hpp"

#include <cmath>

Pieciokat::Pieciokat(int bok) noexcept(true) {
    this->bok = bok;
}

double Pieciokat::Pole() noexcept(true) {
    return sqrt(25 + 10 * sqrt(5)) * ((bok * bok) / 4);
}

double Pieciokat::Obwod() noexcept(true) {
    return 5 * bok;
}

std::string Pieciokat::Nazwa() noexcept(true) {
    return "Pieciokat";
}